crates/swap_aftermath/src/provider.rs

```diff
@@ -51,7 +51,7 @@ impl SwapProvider for AftermathProvider {
             );
             let tx_response: String = self.provider.request_json(AftermathApi::Tx(tx)).await?;
             data = Some(SwapQuoteData {
-                to: request.to_asset.token_id.clone().unwrap_or_default(),
+                to: "".into(), // to address is encoded in tx data
```

Comment:
let's include if possible? to check make sure correct recipient/contract is used, if available inside the `tx_response`

Reply:
I was planning to do that, need to decode the raw tx first

Reply:
I decoded the tx, it's a programmable tx with a bunch of commands (MoveCall)